Makefile

```diff
@@ -87,9 +94,9 @@ install-bls: build-bls
 ###                                Build Tendermint                        ###
 ###############################################################################
 
-build:
+build-binary:
```

Comment:
`build` is still there, and working, but it also builds BLS.

We rename here because in the workflow we don't want to build BLS (which is built separately and cached). Previously, `build` was building both.